angular.json

```diff
@@ -46,7 +46,8 @@
               "node_modules/primeng/resources/primeng.min.css",
               "node_modules/@fortawesome/fontawesome-free/css/all.min.css",
               "src/styles.scss",
-              "src/assets/fonts/titillium/stylesheet.css"
+              "src/assets/fonts/titillium/stylesheet.css",
+              "src/assets/fonts/source-sans-pro/stylesheet.css"
```

Comment:
About bundle size, @smadbe In the issue you wrote:

>  load a different css color set (at launch) depending on this parameter

I understood that the theme file (holding variables) had to be loaded **only** when the theme is "active". When checking at the file for "client-demo", the file itself is very lightweight therefore I don't know if the optimization is relevant ([file anchor](https://github.com/France-ioi/AlgoreaFrontend/pull/928/files#diff-23fad3928bf3e71585eb4a6e3b2ff72dae02c3ed5a44d2f0deb1d49f0cb3b1a3R33))

However, here we seem to load 2 fonts (source-sans-pro fonts are 21~34kB **per font**) to use only one in the end. In my opinion we should load one or the other (using CDN ?), how do you see it ? (I didn't find indication in the issue)

Reply:
ok :+1: